$OpenBSD: patch-i3bar_include_util_h,v 1.1 2014/01/11 16:04:43 dcoppa Exp $

commit 0883dfbe14628430caaeefdb00e282a008d30d76
Author: Michael Stapelberg <michael@stapelberg.de>
Date:   Tue Dec 24 10:35:56 2013 +0100

only LOG() the DPI when it changes, DLOG() it otherwise

This avoids flooding stdout every time some text (e.g. a window
decoration) is drawn, yet leaves the message in place when it's actually
relevant (upon DPI changes).

--- i3bar/include/util.h.orig	Sun Dec 22 21:12:41 2013
+++ i3bar/include/util.h	Fri Jan 10 21:41:56 2014
@@ -48,6 +48,9 @@
     } \
 } while (0)
 
+#if defined(DLOG)
+#undef DLOG
+#endif
 /* Use cool logging-macros */
 #define DLOG(fmt, ...) do { \
     if (config.verbose) { \
